packages/client/src/envconfig.ts

```diff
@@ -0,0 +1,525 @@
+import * as fs from 'fs';
```

Comment:
We can't have the `@temporalio/client` package depends on Core. User-side env config stuff will need to go either in the `worker` package, or we'll make a new package for this (e.g. `@temporalio/config`).